src/modules/keyboardmanager/common/Helpers.cpp

```diff
@@ -2,20 +2,6 @@
 #include "Helpers.h"
 #include <sstream>
 
-// Function to split a wstring based on a delimiter and return a vector of split strings
-std::vector<std::wstring> splitwstring(const std::wstring& input, wchar_t delimiter)

```

Comment:
@arjunbalgovind I'm using this method in splitting store[d shortcuts keys](https://github.com/microsoft/PowerToys/blob/afa9821703f9c6106c0440152b12141ae4e77ac3/src/modules/keyboardmanager/common/Shortcut.h#L37).

Reply:
I'll add that function back.